tests/cypress/integration/pdv_hoc_site_spec.js

```diff
@@ -0,0 +1,487 @@
+describe('Test H.O.C.', () => {
+
+  it('Coveo search', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Google Analytics', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Qualtrics feedback', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Classic/V1 release notes (en-US)', () => {
+    cy.visit('en-us/Content/Topics/ReleaseNotes/okta-relnotes.htm')
+
+  })
+
+  it('OIE release notes (en-US)', () => {
+    cy.visit('oie/en-us/Content/Topics/ReleaseNotes/oie-relnotes.htm')
+
```

Comment:
same here, let's add something to check that we loaded the release notes.
- Verify that the title in the page is "Okta Identity Engine Release Notes"
- Verify that we have 3 visible tiles/cards

Reply:
Added